molecule/saas/sample_cert/jwt/jwt-key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
no need to have a different (or cloned) set of keys for each test scenario. Put the keys in one folder that is used by all three test scenarios.